db_release_scripts/mosip_master/sql/1.2.0_master-scripts_release.sql

```diff
@@ -245,6 +247,9 @@ ALTER TABLE master.batch_job_execution_params ALTER COLUMN string_val TYPE varch
 ALTER TABLE master.blocklisted_words DROP CONSTRAINT IF EXISTS pk_blwrd_code;
 ALTER TABLE master.blocklisted_words ALTER COLUMN lang_code DROP NOT NULL;
 ALTER TABLE master.blocklisted_words ADD CONSTRAINT pk_blwrd_code PRIMARY KEY (word);
+
+INSERT into master.blocklisted_words (word,descr,lang_code,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) SELECT word,descr,lang_code,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes FROM master.blacklisted_words;
```

Comment:
As per this we can see that old table is still kept, rather this could have been a simple table rename. If changed, please take care of the same in the revoke script

Reply:
Done

Reply:
Resolved